draft-ietf-tls-tls13.md

```diff
@@ -3059,7 +3059,7 @@ Alert messages ({{alert-protocol}}) MUST NOT be fragmented across records.
            ContentType type;
            ProtocolVersion legacy_record_version = { 3, 1 };    /* TLS v1.x */
            uint16 length;
-           opaque fragment[TLSPlaintext.length];
+           opaque fragment[length];
```

Comment:
As commented in f4004cf8dc10f362a8e2657f31559b95ebf84b0d, "opaque encrypted_record[length]" is already used in TLSCiphertext. I'd suggest to fix one of them.